docs/buildah-build.1.md

```diff
@@ -708,6 +708,8 @@ Valid _type_ values are:
 If no type is specified, the value defaults to **local**.
 Alternatively, instead of a comma-separated sequence, the value of **--output** can be just a destination (in the `**dest** format) (e.g. `--output some-path`, `--output -`) where `--output some-path` is treated as if **type=local** and `--output -` is treated as if **type=tar**.
 
+Note: The **--tag** option can also be used to change the file image format to supported `containers-transports(5)`.
```

Comment:
I think it might be worth adding an example here to make it more clear. For example, `podman build --tag=oci-archive:./foo.ociarchive .` will build a container and store it in oci-archive format in the file `./foo.ociarchive`.

I know this might make the docs a little more ugly, but might be worth the tradeoff.